### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by OlehHinko**

### CODE REVIEW

**Comment by akorunska** [comment]
Please, add description to the PR with brief description of the addressed issues or improvements implemented. This way review will be much more effective.

File: src/components/layout/UserWalletAddress.js [hunk 1]
```diff
@@ -77,15 +76,17 @@ class UserWalletAddress extends Component {
...
-										}
-									</MyContext.Consumer>
-								</Button>
+								<Tooltip placement="left" title={account.label}>
+									<Button block type="primary" onClick={() => this.handleLogin(account.address)}>
+										<MyContext.Consumer>
+											{activeBreakPoint =>
+												activeBreakPoint !== "sm" && activeBreakPoint !== "xs"
+													? account.address
+													: `${account.address.slice(0, 5)}...${account.address.slice(-5)}`
```
Comment (akorunska) [inline src/components/layout/UserWalletAddress.js]: Please, use special address trimming function from utils.
https://github.com/OnyxPay/onyxpay.co-frontend/blob/d52a1f017b0a65d4c67482e00021a29b2ec9a71d/src/utils/index.js#L92

File: src/components/modals/ChoosePerformer/index.js [hunk 2]
```diff
@@ -79,9 +79,9 @@ class ChoosePerformer extends Component {
 				if (openedRequestData.type === "withdraw") {
 					showNotification({
 						type: "success",
-						msg: "You have successfully chosen the Agent",
+						msg: "You have successfully chosen the agent",
 						desc:
-							"Next, you should wait until fiat money is coming, and then finalize the request by clicking on the 'Perform' button",
+							"Please, wait for the agent to hand over fiat money or send it to you settlement account, and then finalize the request by clicking on the “Perform Withdraw” button.",
```
Comment (akorunska) [inline src/components/modals/ChoosePerformer/index.js]: "Please, wait for the agent to hand over fiat money or send it to you settlement account.."
your settlement account?

**Comment by OlehHinko** [comment]
https://scaddr.apriorit.com/jira/browse/ONXP-1608
https://scaddr.apriorit.com/jira/browse/ONXP-1609
https://scaddr.apriorit.com/jira/browse/ONXP-1610
https://scaddr.apriorit.com/jira/browse/ONXP-1611
https://scaddr.apriorit.com/jira/browse/ONXP-1612

**Pull request merged**